### PR TITLE
Support EAM profile

### PR DESCRIPTION
**Pull request by gcask**
Added auto-selection of an EAM profile if it's present.

### CODE REVIEW

File: DCS-SR-Client/UI/ClientWindow/MainWindow.xaml.cs [hunk 1]
```diff
@@ -962,34 +962,50 @@ private void Connect()
...
                                         new ThreadStart(() =>
                                         {
-                                            //Handle Aircraft Name - find matching profile and select if you can
-                                            name = Regex.Replace(name.Trim().ToLower(), "[^a-zA-Z0-9]", "");
-                                            //add one to seat so seat_2 is copilot
-                                            var nameSeat = $"_{seat + 1}";
-
-                                            foreach (var profileName in _globalSettings.ProfileSettingsStore
-                                                         .ProfileNames)
+                                            if (!ClientState.ExternalAWACSModeConnected)
```
Comment (gcask) [inline DCS-SR-Client/UI/ClientWindow/MainWindow.xaml.cs:965]: This effectively makes the EAM mode priority.
Alternatively, we could extend the logic to use the EAM profile only if:
* We're in EAM mode,
* AND an EAM.cfg was found.

Comment (ciribob) [reply DCS-SR-Client/UI/ClientWindow/MainWindow.xaml.cs:965]: Sorry this has taken a while - im going to change this slightly to just set the aircraft name as EAM if you're in EAM mode (so it just works) and if there is no aircraft set - have a GCI profile

Comment (ciribob) [reply DCS-SR-Client/UI/ClientWindow/MainWindow.xaml.cs:965]: Ok... embarassingly I wrote it like that years ago and forgot - a profile name of External AWACS would just work.... but i'll change it to EAM 😅  Sorry!

**Comment by ciribob** [comment]
Thanks for this - as per the comment turns out it always worked 😅 sorry about this and thanks for the PR!

**Pull request closed without merging**